### PR TITLE
Add a recipe for glueviz

### PR DESCRIPTION
**Pull request by astrofrog**
As suggested by @ccordoba12 in https://github.com/ContinuumIO/anaconda-issues/issues/785

@ccordoba12 - glueviz is currently available in the default conda channel, so is there anything I need to do here to make sure that the package that will be produced by this feedstock also gets used for the default conda channel?

I'm suspecting that the tests may currently fail on Travis Linux because I normally need to set up the XVFB since there are some GUI tests (but let's see)

### CODE REVIEW

**Comment by astrofrog** [comment]
@pelson - ``conda smithy recipe-lint`` is saying the build/number section is missing, but I've included it as in other recipes, do you know what could be causing this?

**Comment by astrofrog** [comment]
Ignore me, I found the issue!

File: recipes/glueviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{% set version = "0.7.3" %}
+
+package:
+  name: glueviz
+  version: {{version}}
+
+source:
+  fn: glueviz-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/6d/03/cbef6e6f80112dfc4edb4f1d6a419ea9b159511f0828b94f1bfd730f7cdb/glueviz-{{version}}.tar.gz
```
Comment (ocefpaf) [inline recipes/glueviz/meta.yaml]: Can you use https://pypi.io/packages/source/g/glueviz/glueviz-0.7.3.tar.gz instead?

File: recipes/glueviz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+app:
+  entry: glue
+  icon: logo.png
+  summary: Multi-dimensional linked data exploration
+  type: desk
+
+about:
+  home: http://glueviz.org
+  license: BSD
```
Comment (ocefpaf) [inline recipes/glueviz/meta.yaml]: Do you know which BSD version? 2 or 3-Clause?

Comment (astrofrog) [inline recipes/glueviz/meta.yaml]: 3-clause - is there a certain syntax to write this?

Comment (ocefpaf) [inline recipes/glueviz/meta.yaml]: I like to write that as `BSD 3-Clause`.

Comment (astrofrog) [inline recipes/glueviz/meta.yaml]: Thanks, fixed!

**Comment by ocefpaf** [comment]
Linux is failing with:

```
+ python -c 'import glue; glue.test()'
: cannot connect to X server 
```

Not sure if there a trick to fool Circle and start X for the tests, but you don't really need to run the whole test suit here. We are OK with simple test that check if the package is "working."

**Comment by astrofrog** [comment]
@ocefpaf - I agree that we don't need to run the full test suite, so I've removed that.

**Comment by astrofrog** [comment]
Before merging this though, I'd like to make sure I understand the implications for how this relates to the glueviz package in the default conda channel, so I wonder if @ccordoba12 can clarify?

**Comment by msarahan** [comment]
Please see comment in https://github.com/ContinuumIO/anaconda-issues/issues/785#issuecomment-218746821

Is there a way to run some tests? I understand the x server is something we don't want to get into, but are there other meaningful things to test?

**Comment by ocefpaf** [comment]
@astrofrog this ready to go. Let me know if https://github.com/conda-forge/staged-recipes/pull/592#issuecomment-218749067 answered your question.

**Comment by astrofrog** [comment]
@msarahan - there is no way to run a subset of the non-GUI tests currently, but I've opened an issue for this (https://github.com/glue-viz/glue/issues/985) and will make sure that I update the recipe once this is in a release.

**Comment by msarahan** [comment]
Cool, thanks.  Merging.

**Comment by ccordoba12** [comment]
Sorry for being late in answering you @astrofrog, but as @msarahan said, conda-forge doesn't supersede the default channels but it's really useful to keep your package always updated :-)

**Comment by jakirkham** [comment]
Trying to re-run the CI with `master` to see if there was just a GitHub API or other networking issue with conversion to a feedstock. Please keep an eye out to see if this fixes or not and report here ( https://github.com/conda-forge/staged-recipes/issues/594 ).

**Comment by jakirkham** [comment]
No dice. 🎲 Maybe @pelson will know better why things are failing here.

**Comment by pelson** [comment]
> No dice. 🎲 Maybe @pelson will know better why things are failing here.

Sorry about that. A release of conda-smithy (>0.9,<=0.10.2) broke a few things. Looks like everything is back up and running.

**Comment by astrofrog** [comment]
Thanks @pelson!

**Comment by pelson** [comment]
P.S. Welcome to conda-forge @astrofrog - great to see you on here!

**Pull request merged**